packages/store/mud.config.ts

```diff
@@ -12,6 +12,7 @@ export default mudConfig({
       },
       schema: {
         tableName: "string",
+        abiEncodedKeyNames: "bytes",
         abiEncodedFieldNames: "bytes",
       },
```

Comment:
I think we'll need to update network decoding to handle this:
https://github.com/latticexyz/mud/blob/dd8413340c429ec28f909bbb1b96fd117cf23a5e/packages/network/src/v2/schemas/tableMetadata.ts#L62

Reply:
good spot. with MODE too this is quite involved.